src/index.js

```diff
@@ -49,6 +49,11 @@ const metrics = {
 		name: 'pagespeed_performance_score',
 		help: "Performance Score",
 		labelNames: [ 'page' ]
+	}),
+	accessibility_score: new Prometheus.Gauge({
+		name: 'accessibility_performance_score',
```

Comment:
Would make sense to prefix with pagespeed so you can find it easily in prometheus?

```suggestion
		name: 'pagespeed_accessibility_performance_score',
```